.github/workflows/release.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         with:
```

Comment:
Very weird they just break API like this, since we're using a pinned version of goreleaser/goreleaser-action@v3